docs/intro/install.rst

```diff
@@ -14,12 +14,16 @@ If you're using `Anaconda`_ or `Miniconda`_, you can install the package from
 the `conda-forge`_ channel, which has up-to-date packages for Linux, Windows
```

Comment:
Using the `highlight` directive may make sense in this document to minimize changes.